[FEATURE] #99834 - Add two PSR-14 events around auto-create redirects (#2772)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/Events/Events/Redirects/ModifyAutoCreateRedirectRecordBeforePersistingEvent.rst

..  include:: /Includes.rst.txt
..  index:: Events; ModifyAutoCreateRedirectRecordBeforePersistingEvent
..  _ModifyAutoCreateRedirectRecordBeforePersistingEvent:

===================================================
ModifyAutoCreateRedirectRecordBeforePersistingEvent
===================================================

..  versionadded:: 12.3

The PSR-14 event
:php:`\TYPO3\CMS\Redirects\Event\ModifyAutoCreateRedirectRecordBeforePersistingEvent`

> allows extension authors to modify the redirect record before it is persisted to

Reviewer: ```suggestion
allows extensions to modify the redirect record before it is persisted to
```

the database. This can be used to change values according to circumstances, such

> as, for example, different sub-tree settings that are not covered by the Core

Reviewer: ```suggestion
as different sub-tree settings that are not covered by the Core
```

site configuration. Another use case could be to write data to additional
:sql:`sys_redirect` columns added by a custom extension for later use.

..  note::

>     To handle later updates or react on manual created redirects in the backend

Reviewer: ```suggestion
    To handle updates or react on manual created redirects in the backend
```
